examples/features/csm_observability/client/main.go

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/examples/features/proto/echo"
+	"google.golang.org/grpc/stats/opentelemetry"
+	"google.golang.org/grpc/stats/opentelemetry/csm"
+	_ "google.golang.org/grpc/xds" // To install the xds resolvers and balancers.
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.opentelemetry.io/otel/exporters/prometheus"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+var (
+	addr     = flag.String("addr", "xds:///helloworld:50051", "the server address to connect to")
+	promAddr = flag.String("promAddr", ":9464", "the Prometheus exporter endpoint")
```

Comment:
The command line flag name should be `prom_addr` or `prom-addr` if other languages are using the latter.

Also (optional), maybe since you have a `prom_addr`, the other flag could be `server_addr`

Reply:
I am the only one making this configurable. C++ hardcodes this, but Doug really wanted this to be a knob for extensibility. I'll switch to prom_addr and server_addr.